src/roi/Roi.ts

```diff
@@ -31,17 +37,47 @@ export class Roi {
   /**
    * Surface of the ROI.
    */
+
   public surface: number;
 
+  public maxColumn: number;
+  public maxRow: number;
```

Comment:
I don't think those 2 properties should be defined. Currently they are used once in the full code and they can easily be calculated on the fly.

Even if there is a calculation in the end condition `for (let i=0; i<roi.origin.column+roi.width-1; i++)` V8 will likely optimize this and will do only once the calculation (https://jsbench.me/ailfjqjx7r/1)